stdlib/tkinter/__init__.pyi

```diff
@@ -3166,12 +3196,45 @@ class Text(Widget, XView, YView):
     def tag_ranges(self, tagName: str) -> tuple[_tkinter.Tcl_Obj, ...]: ...
     # tag_remove and tag_delete are different
     def tag_remove(self, tagName: str, index1: _TextIndex, index2: _TextIndex | None = None) -> None: ...
-    # TODO: window_* methods
-    def window_cget(self, index, option): ...
-    def window_configure(self, index, cnf: Incomplete | None = None, **kw): ...
+    @overload
+    def window_cget(self, index: _TextIndex, option: Literal["padx", "pady"]) -> int: ...
+    @overload
+    def window_cget(self, index: _TextIndex, option: Literal["stretch"]) -> Literal[0, 1]: ...
```

Comment:
This is technically wrong, but it will be more convenient and practical to just use `bool`.

```suggestion
    def window_cget(self, index: _TextIndex, option: Literal["stretch"]) -> bool: ...  # actually returns Literal[0, 1]
```

For example, you might want to copy a stretch value from one embedded window to another, and you would do it like this:

```python
text.window_configure(index1, stretch=text.window_cget(index2, 'stretch'))
```

Reply:
An alternative solution is to make `window_configure` accept `bool | Literal[0, 1]`, that way this would still work while accurately describing the runtime behaviour. If it was annotated as returning bool I could imagine users doing `blah is True`. Though this would be probably inconsistent with the rest of the module..

Reply:
Indeed, we could change the `bool` handling in a separate PR.